TransIT.DAL/Models/Entities/IssueLog.cs

```diff
@@ -31,6 +31,8 @@ public IssueLog()
         public virtual State NewState { get; set; }
         public virtual State OldState { get; set; }
         public virtual Supplier Supplier { get; set; }
+
         public virtual ICollection<Document> Document { get; set; }
+        public virtual ICollection<IssueLogPart> IssueLogPart { get; set; }
```

Comment:
this property name for collection should not be IssueLogPart, should be IssueLogPart**s**